src/rgw/rgw_sal_motr.cc

```diff
@@ -3744,22 +3737,24 @@ int MotrMultipartUpload::list_parts(const DoutPrefixProvider *dpp, CephContext *
     if(ret_rc < 0)
       return ret_rc;
 
-    key_name = ent.key.name + "[" + ent.key.instance + "]";
-
-    // fetch the version-id in case of null version-id
-    if(ent.key.instance == "null")
+    if(!ent.is_delete_marker())
     {
-      ret_rc = mobj_ver->fetch_null_obj_reference(dpp, key_name);
-      if(ret_rc < 0) {
-        ldpp_dout(dpp, 0) << __func__ << " : failed to get null object reference, ret_rc : "<< ret_rc << dendl;
-        return ret_rc;
-      }
-    }
+      key_name = ent.key.name + "[" + ent.key.instance + "]";
```

Comment:
pls add example of key_name as code references.
e.g. // object1[sdasdafdffsdfd]

Reply:
done